include/image-commands.mk

```diff
@@ -355,6 +355,21 @@ define Build/tplink-v2-image
 	rm -rf $@.new
 endef
 
+# Overwrite the first 160 bytes of the image with wrgg header.
+# Make room for this header by building the image with e.g. "pad-extra 160 | append-kernel | ..."
+define Build/wrgg-image
```

Comment:
Hm, are you planning on adding more D-Link devices? Or will you convert the existing gemini (DIR-685) and ar71xx (dap-2695-a1) device to this helper? Because it looks a bit out of place since I think, you could have gotten away with just adding this right after "Build/SenaoFW" in the target/linux/ipq40xx/image/Makefile and have somebody else worry about making it all work with the gemini and ar71xx device.

Reply:
I had some intention of doing that, but I moved the function to the Makefile for now. If I or someone else gets around to unifying the mkwrgg functions, s/he move it again.